spec/features/collection_drafts/breadcrumbs_and_header_spec.rb

```diff
@@ -31,7 +31,7 @@
       visit collection_draft_path(@draft)
     end
 
-    it 'displays the short name' do
+    it 'displays the short name within in breadcrumbs' do
```

Comment:
really minor, but I think there's a typo. the 'in' after 'within' should be changed or removed.